recipes/natural_language_processing/function-calling-nodejs/app/Containerfile

```diff
@@ -0,0 +1,65 @@
+# # Install dependencies only when needed
```

Comment:
Do we need to keep those commented instructions, they make it hard for a newcomer to read and understand the content

Reply:
probably not.  I think i was going to remove them and forgot.  let me update